docs/asset-store/docs/08-01-gateway-mode-set-up.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Set Minio to the Google Cloud Storage Gateway mode
+type: Tutorials
+---
+
+By default, you install Kyma with the Asset Store in Minio stand-alone mode. This tutorial shows how to set Minio to the Google Cloud Storage (GCS) Gateway mode using an [override](/root/kyma/#tutorials-helm-overrides-for-kyma-installation).
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [gcloud](https://cloud.google.com/sdk/gcloud/)
+- [Google Cloud Platform (GCP)](https://cloud.google.com) project
+
+## Steps
```

Comment:
I would love to have a clear separation of the steps, which ones are about setting up a credentials in GCP and which are about configuring Kyma and triggering installation.

we need to remember that there are also pro users that will not need gcp instructions and will just have a JSON in their hands already. It will be easier for them to navigate to proper kyma config instruction rather than read all the steps

Reply:
I had it that way and changed it when I was removing toggles. OK.